docs/docs/API-Reference/api-reference-api-examples.md

```diff
@@ -48,12 +48,21 @@ The Langflow port number is set in the `LANGFLOW_PORT` [environment variable](/e
 
 ### Authentication
 
-Your [Langflow deployment's authentication settings](/configuration-authentication) determine whether Langflow API requests require explicit authentication with a Langflow API key.
+<details closed>
```

Comment:
Please see my suggestion on the other page that has a similar details block. The "1.5 and later" info should be outside details.